cfgov/unprocessed/css/pages/ask.less

```diff
@@ -30,4 +30,45 @@
     ol:last-child {
         margin-bottom: 0 !important;
     } 
+}
+
+.ask-landing-page {
+    .content_main {
```

Comment:
If the `content_main` class were to be removed and layout classes left, would this be necessary? Thinking back to the  home page, we didn't use `content_main` for the top half specifically because of the padding it adds.